NEWS

```diff
@@ -11,9 +11,143 @@
+* Fixed endpoint reconfiguration error due to wrong endpoint configuration locality assignment
+#### RDMA CORE (IB, ROCE, etc.)
+* Fixed performance degradation when KSM with atomic access fails repeatedly during rkey_pack phase of rendezvous protocol
+* Fixed invalid access error to strict-order key with offset
+#### GPU (CUDA, ROCM)
+* Fixed CUDA IPC performance degradation after libnuma removal
+#### Build
+* Updated tags for CUDA 11 images, as older versions were removed from Dockerhub
+
 ## 1.15.0-rc1 (May 10, 2023)
-TBD
+### Features:
+#### UCP
+* Added 2-stage pipeline protocol in the new protocol infrastructure
+* Added reset and abort functionality of Rendezvous protocols in the new infrastructure
+* Added zero-copy Rendezvous data send protocol in the new infrastructure
+* Added support for user memory handle in the new protocol infrastructure
+* Added option to force ODP registration for certain memory types
+* Enabled lock free memory region deregistration
+* Updated allow/deny transport list feature to control auxiliary transport selection
+* Multiple performance improvements of the new protocol infrastructure
+* Multiple improvements in error and debug messages
+#### UCT
+* Split UCT_MD_MKEY_PACK_FLAG_INVALIDATE into two flags for RMA and AMO
+* Added put_zcopy and get_zcopy scheme support for self transport
+* Added base implementation of is_reachable_v2 API using intra/inter flag
+#### RDMA CORE (IB, ROCE, etc.)
+* Added option to control CQE zipping per CQ RX/TX direction
+* Added option to specify how DCI select port under RoCE LAG
+* Added hw_dcs to the list of policies to select DCI by an endpoint
+* Removed implicit on-demand paging
+* Added option to set RoCE lag dct port for response under queue affinity mode
+* Improved IB memlock limit logging
+#### UCS
+* Added ucs_string_buffer_rbrk() to split token
+#### GPU (CUDA, ROCM)
+* Added support for atomic reply_buffer on GPU memory
+* Added system device information for AMD GPUs
+* Improved performance estimation of gdr_copy transport
+* Added a simplistic implementation of performance estimation of cuda_ipc transport
+* Improved performance estimation of cuda_ipc on Hopper architecture
+* Added rcache parameters for rocm transports
+* Introduced dmabuf support for rocm transports
+* Implemented asynchronous progress for the zcopy operations in the rocm_copy transport
+* Added option to enable using cross-device dmabuf file descriptor for rocm
+#### Java
+* Added Java bindings for exported memh feature
+#### Tests
+* Added a rocm docker container for testing
+* Added option to send client Id in IODEMO test
+* Added support for repeating connection to the same server in IODEMO test
+* Added receive callback for receiving FIN message in UCX examples
+#### Tools
+* Added user-side memcpy option for AM benchmarks in ucx_perftest
+* Added wireshark LUA dissectors for some UCX protocols
+#### Build
+* Added a separate xpmem deb subpackage
+* Added ARM-based artifacts.
```

Comment:
please remove the `.` in the end (for consistency)

Reply:
Removed